spec/active_model_serializers/cancan/associations_spec.rb

```diff
@@ -100,11 +100,11 @@ def initialize(user)
       end
     end
 
-    it 'should filter unauthorized records', focus: true do
+    xit 'should filter unauthorized records', focus: true do
       expect(CategorySerializer.new(category, scope: user).serializable_hash[:projects].length).to eq(1)
     end
 
-    it 'should nil out unauthorized has_one records' do
+    xit 'should nil out unauthorized has_one records' do
```

Comment:
why disabling those tests

Reply:
I think I was playing around with specs and commented a bunch of them.
I'll double-check those specs because they started to fail after updating cancancan to 3.0

Reply:
Due to update with active_model_serializers, part of the code from this gem no longer works.
That's why specs were failing. Since we don't use a feature from ams_cccc I believe the effort to fix the code is not worth it, so the unused broken code was removed.